relationships/synthesis/APM-APPLICATION-to-INFRA-MYSQL.yml

```diff
@@ -0,0 +1,30 @@
+relationships:
+  - name: apmApplicationCallsInfraRedisInstance
```

Comment:
name should be unique so this probably should be about mysql

Reply:
just seeing this. 🤦  C&P error thanks for solving it